docker-compose.yml

```diff
@@ -1,5 +1,46 @@
 version: '3'
 services:
+  pumba:
+    container_name: pumba
+    image: alpine-docker
+    user: "root"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    # command:
+      # - /bin/sh
+      # - -c
+      # - |
+      #   pumba netem --tc-image gaiadocker/iproute2 --duration 1m loss -p 50 re2:validator-node-2
+    networks:
+      tx-network:
+        ipv4_address: 10.6.0.20
+      block-network:
+        ipv4_address: 10.5.0.20        
+
+  tx-node-1:
+    container_name: tx-node-1
+    image: "node:8"
+    user: "node"
+    working_dir: /home/node/app
+    environment:
+      - NODE_ENV=production
+    volumes:
+      - /Users/triet/centrality/plug-load-test/:/home/node/app
```

Comment:
This path should be changed so that it can 'run anywhere'

Reply:
fixed to mount current directory